Templates/DefaultProject/Template/.command_settings

```diff
@@ -0,0 +1,6 @@
+[export_project]
+project.build.config = release
+archive.output.format = zip
+seedlist.paths = AssetBundling/SeedLists/*.seed
+option.build.assets = True
+option.build.tools = True
```

Comment:
For better support with installer builds, I think we need to add a default value for the monolithic builds:
```suggestion
option.build.monolithic= True
```